imagetest/test_suites/image_validation/linux_lincense_test.go

```diff
@@ -0,0 +1,67 @@
+package imagevalidation
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"unicode"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+	"google.golang.org/api/compute/v1"
```

Comment:
you can't use APIs from tests. the test VM will not be guaranteed to have permission to call this API.